src/js/fluxcontroller.js

```diff
@@ -927,7 +927,7 @@ define(function (require, exports, module) {
                 debouncedName = name + DEBOUNCED_ACTION_SUFFIX,
                 synchronizedAction = this._synchronize(namespace, module, name),
                 throttledAction = synchronization.throttle(synchronizedAction),
-                debouncedAction = synchronization.debounce(synchronizedAction);
+                debouncedAction = _.debounce(synchronizedAction);
```

Comment:
Also, a quick grep did not turn up any instances in which we're actually using those actions.